critiquebrainz/frontend/messages.pot

```diff
@@ -211,7 +211,7 @@ msgid "Please sign in to access this page."
 msgstr ""
 
 #: critiquebrainz/frontend/login/__init__.py:46
-msgid "You must be an administrator to view this page."
+msgid "You must be a moderator to view this page."
```

Comment:
Thanks for the PR! :)
You don't need to modify the `.pot` and `.po` files as they are generated using a script. 
Make sure you revert those changes.